script/BaseScript.s.sol

```diff
@@ -456,51 +358,13 @@ contract BaseScript is BaseData, Utils {
         return (size > 0);
     }
 
-    function _predictTimelockController(
```

Comment:
As mentioned above predictions are no longer required since we use `CREATE3Factory` directly